gwdetchar/lasso/core.py

```diff
@@ -42,18 +42,32 @@ def find_outliers(ts, N=5):
         data to find outliers within
 
     N : `float`, optional
-        number of standard deviations to consider an outlier, default: 5
+        if `method='s'`: number of standard deviations to consider an outlier
+        if `method='pf'`: percentile range limit to consider an outlier
+        default value for both methods:
+    S : `String`, optional
```

Comment:
This part of the docstring should reflect the name of the keyword argument in the function spec, something like:

```python
    method : `str`, optional
        outlier identification method to use, must be either `'s'` (standard
        deviation method) or `'pf'` (percentile range method),
        default: `'s'`
```